app/assets/stylesheets/recipe_finder.css.scss

```diff
@@ -57,12 +57,12 @@
   background: $color-airbnb-red;
   font-family: $font-airbnb;
   // box-shadow: 1px 1px 5px $color-airbnb-font;
-  border: 1px solid $color-airbnb-grey;
+  border: 1px solid $color-airbnb-red;
```

Comment:
agree, will do at later stage..... because right now I'm not familiar with color design enough to determine what is a secondary or primary color at the moment